website/docs/language/modules/sources.mdx

```diff
@@ -262,8 +262,16 @@ to reduce the time taken to retrieve the remote repository.
 
 The `depth` URL argument corresponds to
 [the `--depth` argument to `git clone`](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt),
-telling Git to create a shallow clone with the history truncated to only
-the specified number of commits.
+instructs the Git to create a shallow clone with the history truncated to only
+the specified number of commits. 
+Because Terraform only uses the most recent selected commit to find the source
+code of your specified module, it is not typically useful to set `depth`
+to any value other than `1`.
```

Comment:
```suggestion
which instructs Git to create a shallow clone that includes
only the specified number of commits in the history. 
Setting `depth` to `1` is suitable for most cases. This is because Terraform only uses the most recently selected commit to find the source.
```

Reply:
yeah, make sense